src/unittest/test.cpp

```diff
@@ -248,7 +251,10 @@ bool run_tests()
 		<< num_total_tests_run << " failed individual tests)." << std::endl
 		<< "    Testing took " << tdiff << "ms total." << std::endl
 		<< "++++++++++++++++++++++++++++++++++++++++"
-		<< "++++++++++++++++++++++++++++++++++++++++" << std::endl;
+		<< "++++++++++++++++++++++++++++++++++++++++" << std::endl
+	        << "Catch test results: " << std::endl;
+	auto catch_test_failures = Catch::Session().run();
+	num_modules_failed += catch_test_failures;
```

Comment:
please move this up so it is
1. taken into account for time elapsed
2. somehow added to `num_modules_failed` (in the output)

Reply:
Do you mean `num_total_tests_failed`? The reason I didn't include it in that count was because Catch2 prints its own success and failure information, and I couldn't find a way to access the number of successes. How would you prefer to handle that?